LayoutTests/svg/filters/filter-specified-on-svg-root.html

```diff
@@ -0,0 +1,14 @@
+<p>This test passes if the rectangle only contains a single shade of green.</p>
+
+<span>
+  <svg filter="url(#fade)" width="100" height="100">
+    <filter id="fade">
+      <feComponentTransfer>
+        <feFuncA type="table" tableValues="0 0.6"/>
+      </feComponentTransfer>
+    </filter>
+    <rect width="100" height="100" fill="green" />
+  </svg><svg width="100" height="100">
+    <rect width="100" height="100" fill="green" fill-opacity="0.6" />
+  </svg>
```

Comment:
These two lines seems to be left by mistake.

Reply:
Oh I think I was confused by this markup. I did not see the opening <svg> tag.
```
</svg><svg width="100" height="100">
```

Reply:
Yeah, the ending `</svg>` of the first SVG has to be on the same line as the opening `<svg>` of the second SVG, so they'd be next to each other without gaps. Or there might a trick to achieve the same effect.